file-systems/unixfs: fix default leaf format. (#724)
ipfs/ipfs-docs: docs/concepts/file-systems.md

The leaf format takes two format options, UnixFS leaves and raw leaves:

> - The UnixFS leaves format (default) adds a data wrapper on newly added objects to produce UnixFS leaves with additional data sizes. This wrapper is used to determine whether newly added objects are files or directories.

Reviewer: ```suggestion
- The UnixFS leaves format (default for CIDv0) adds a data wrapper on newly added objects to produce UnixFS leaves with additional data sizes. This wrapper is used to determine whether newly added objects are files or directories.
```

> - The UnixFS leaves format (default for CIDv0) adds a data wrapper on newly added objects to produce UnixFS leaves with additional data sizes. This wrapper is used to determine whether newly added objects are files or directories.

Reviewer 2: ```suggestion
- The UnixFS leaves format adds a data wrapper on newly added objects to produce UnixFS leaves with additional data sizes. This wrapper is used to determine whether newly added objects are files or directories. This format is the default for CIDv0.
```

> - The raw leaves format (experimental, likely future default) on IPFS where nodes output from chunking will be raw data from the file with a CID type of 'raw'. This is mainly configured for backward compatibility with formats that used a UnixFS Data object.

Reviewer: ```suggestion
- The raw leaves format (default for CIDv1 created with `ipfs add --cid-version 1`, soon to become the global default) on IPFS where nodes output from chunking will be raw data from the file with a CID codec of 'raw'. This is mainly configured for backward compatibility with formats that used a UnixFS Data object.
```

> - The raw leaves format (default for CIDv1 created with `ipfs add --cid-version 1`, soon to become the global default) on IPFS where nodes output from chunking will be raw data from the file with a CID codec of 'raw'. This is mainly configured for backward compatibility with formats that used a UnixFS Data object.

Reviewer 2: ```suggestion
- The raw leaves format on IPFS where nodes output from chunking will be raw data from the file with a CID codec of 'raw'. This is mainly configured for backward compatibility with formats that used a UnixFS Data object. This format is the default for CIDv1 created with `ipfs add --cid-version 1`, soon to become the global default.
```
